blog/en/blog/2023/02/09/secure-apisix-admin.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Securing Admin Access to Apache APISIX
+authors:
+  - name: Nicolas Fränkel
+    title: Author
+    url: https://github.com/nfrankel
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - API gateway security
+  - Admin API
+  - how to secure API gateway
+description: >
+  API Gateways are critical components in one's infrastructure. If an attacker could change the configuration of routes, they could direct traffic to their infrastructure. Consequences could range from data theft to financial losses. Worse, data theft could only be noticed after a long time by mirroring the load. Hence, protecting your API Gateway is of utmost importance
+tags: [Ecosystem]
+image: https://blog.frankel.ch/assets/resources/secure-apisix-admin/security-g85f9e284f.jpg
```

Comment:
Could you use a different link for the cover pic? Since the current one does not show up properly on the summary page. 

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/36651058/217803305-c2fde64d-3f9e-43a1-8df7-75eade76bce7.png">

Reply:
Please take the picture and upload it on the apiseven cloud. Or let me know how to do it, it's a recurring issue.

Reply:
@juzhiyuan @SkyeYoung  would you guys be able to do knowledge transfer of picture conversions via pm?

Reply:
Sure, here it is: https://static.apiseven.com/uploads/2023/02/09/KMA4XPZD_security-g85f9e284f.jpg